index.html

```diff
@@ -8,17 +8,64 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <h1>Awesome books</h1>
-    <div class="book-list">
+    <nav class="nav_wrapper">
+        <div class="menu_wrapper">
+            <div class="logo">Awesome Books</div>
+            <ul class="menu_list_wrapper">
+                <li class="list"><a class="menue_txt" href="#bookList">List</a></li>
+                <li class="list"><a class="menue_txt" href="#addNew">Add new</a></li>
+                <li class="list"><a class="menue_txt" id="cont_txt" href="#contactInfo">Contact</a></li>
+            </ul>
+        </div>
+    </nav>
+    <a name="bookList"></a>
+    
+    <div class="list_page">
+        <span id="nav_span">
+        
+        </span>
+        
+        <h1 class="awsome-title">Awesome books</h1>
+        <div class="book-list">
 
 
+        </div>
+        <div class="copy_right">Copyright .....</div>
+    </div>  
+    
+    <div class="new_list">
+        
+        <a name="addNew"></a>
+        <span>
+            
+        </span>
+        <form>
+            <h2>Add a new book</h2>
+            <input type="text" name="text" id="txt" placeholder="Title" required><br><br>
+            <input type="text" name="text" id="txtOne" placeholder="Author" required><br><br>
+            <button type="submit" class="add">Add</button>
+        </form>
+        <div class="copy_right">Copyright .....</div>
     </div>
-    <form>
-        <h2>Add a new book</h2>
-        <input type="text" name="text" id="txt" placeholder="Title" required><br><br>
-        <input type="text" name="text" id="txtOne" placeholder="Author" required><br><br>
-        <button type="submit" class="add">Add</button>
-    </form>
+
+    <div class="contact_page">
+        
+        <a name="contactInfo"></a>
+        <span>
+    
+        </span>
+        <h1 class="contact-info">Contact information</h1>
+        <p class="contact">If you have any question about the services that we give do contact as with the following contact info.</p>
+        <ul class="contact-list">
+            <li class="con_list">* Our e-mail : addisalemseifu@gmail.com</li>
+            <li class="con_list">* Our phone no : +2519 10528787</li>
+            <li class="con_list">* Our Address : Bole 22, AddisAbaba, Ethiopia</li>
+        </ul>
+        <div class="copy_right">Copyright .....</div>
+    </div>
```

Comment:
- [ ] It would be better if the only shows the section that is clicked. Currently, the app shows all the sections. You can use CSS to display and remove the sections. 

![Capture](https://user-images.githubusercontent.com/64173189/139483750-a32ecc17-879f-49f9-836a-783439e30466.JPG)

😄 💪